snap/snapcraft.yaml

```diff
@@ -162,15 +150,15 @@ parts:
       echo "Starting at $(date)"
       START_TIME=$(date +%s)
 
-      CPU_COUNT=$(lscpu --parse=CPU | grep -v "^#" | wc -l)
-
-      export DESTDIR="$(readlink --canonicalize ${SNAPCRAFT_PART_INSTALL})"
+      DESTDIR="${SNAPCRAFT_PART_INSTALL}"
 
       ./otp_build autoconf
       ./configure --prefix=/usr
 
-      make -j ${CPU_COUNT}
-      make install
+      make -j ${SNAPCRAFT_PARALLEL_BUILD_COUNT}
```

Comment:
This line is executed, see

https://github.com/nicolasbock/rabbitmq-server-snap/runs/7997665722?check_suite_focus=true#step:3:2769